sources/HUBComponentWrapper.m

```diff
@@ -282,8 +281,9 @@ - (void)component:(id<HUBComponentWithChildren>)component didStopDisplayingChild
         return;
     }
     
+    HUBComponentWrapper * const childComponent = self.childrenByIndex[@(childIndex)];
```

Comment:
This will break for components that are using normal `UIViews` for their child components instead of creating them through the framework. It's a usecase that we want (and need to - for legacy) support :/

Basically this will be `nil` for those type fo components.